kroma-validator/l2_output_submitter.go

```diff
@@ -181,6 +287,16 @@ func (l *L2OutputSubmitter) repeatSubmitL2Output(ctx context.Context) {
 // If it needs to wait, it will calculate how long the validator should wait and
 // try again after the delay.
 func (l *L2OutputSubmitter) trySubmitL2Output(ctx context.Context) (time.Duration, error) {
+	if l.isValManagerEnabled {
```

Comment:
Instead of `l.isValManagerEnabled`, at first of the `trySubmitL2Output` loop, fetch next output index to submit and use it below logics to separate V1 and V2.